demo/lib/vm.bash

```diff
@@ -328,7 +337,7 @@ vm-sed-file() { # script API
     shift
     for cmd in "$@"; do
         vm-command "sed -E -i \"$cmd\" $file" ||
-            command-error "failed to edit $file with sed commands $@"
+            command-error "failed to edit $file with sed"
```

Comment:
command-error "failed to edit $file with sed command $cmd" ?

Reply:
`command-error` will print the original command (in this case `vm-command` parameter). So I'd avoid repeating the failed sed command in the output.